test/Microsoft.Net.Http.Headers.Tests/CookieHeaderValueTest.cs

```diff
@@ -301,25 +294,31 @@ public void CookieHeaderValue_ParseList_ExcludesInvalidValues(IList<CookieHeader
         [MemberData(nameof(ListWithInvalidCookieHeaderDataSet))]
         public void CookieHeaderValue_TryParseList_ExcludesInvalidValues(IList<CookieHeaderValue> cookies, string[] input)
         {
-            IList<CookieHeaderValue> results;
-            var result = CookieHeaderValue.TryParseList(input, out results);
+            var result = CookieHeaderValue.TryParseList(input, out var results);
             Assert.Equal(cookies, results);
             Assert.Equal(cookies?.Count > 0, result);
         }
 
         [Theory]
         [MemberData(nameof(ListWithInvalidCookieHeaderDataSet))]
-        public void CookieHeaderValue_ParseStrictList_ThrowsForAnyInvalidValues(IList<CookieHeaderValue> cookies, string[] input)
+        public void CookieHeaderValue_ParseStrictList_ThrowsForAnyInvalidValues(
+#pragma warning disable xUnit1026 // Theory methods should use all of their parameters
```

Comment:
Yup. Because we re-use `[MemberData(nameof(ListWithInvalidCookieHeaderDataSet))]` in many tests, this is the quickest solution. You have to have a parameter to match each value in the MemberData, but if it's unused in the test method, xunit.analyzers will issue a build warning. 

Going forward it will be better to factor MemberData into separate collections that match their actual usage, if possible.